src/index.html

```diff
@@ -1,27 +1,401 @@
+        <div class="card__stars">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <div class="card__reviews">Reviews: 5</div>
+        </div>
+        <div class="card__price">
+          <div class="card__price-label">Price:</div>
+          <div class="card__price-amount">$2,199</div>
+        </div>
+        <div class="card__button">
+          <a
+            href="#"
+            class="card__button-link"
+            data-qa="hover"
+          >
+            BUY
+          </a>
+        </div>
+      </div>
+      <div class="card">
+        <img
+          class="card__image"
+          src="./images/imac.jpeg"
+          alt="imac"
+        />
+
+        <div class="card__name">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </div>
+        <div class="card__code">Product code: 195434</div>
+        <div class="card__stars">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <div class="card__reviews">Reviews: 5</div>
+        </div>
+        <div class="card__price">
+          <div class="card__price-label">Price:</div>
+          <div class="card__price-amount">$2,199</div>
+        </div>
+        <div class="card__button">
+          <a
+            href="#"
+            class="card__button-link"
+            data-qa="hover"
+          >
+            BUY
+          </a>
+        </div>
+        <div class="card">
+          <img
+            class="card__image"
+            src="./images/imac.jpeg"
+            alt="imac"
+          />
+
+          <div class="card__name">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </div>
+          <div class="card__code">Product code: 195434</div>
+          <div class="card__stars">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+            <div class="card__reviews">Reviews: 5</div>
+          </div>
+          <div class="card__price">
+            <div class="card__price-label">Price:</div>
+            <div class="card__price-amount">$2,199</div>
+          </div>
+          <div class="card__button">
+            <a
+              href="#"
+              class="card__button-link"
+              data-qa="hover"
+            >
+              BUY
+            </a>
+          </div>
+        </div>
```

Comment:
It seems like there is an extra closing div tag for the card class. This might be causing an issue with the layout. Please ensure that each opening tag has a corresponding closing tag.